NikoBracket/HomeViewController.m

```diff
@@ -12,6 +12,15 @@
 #import "HomeViewCell.h"
 #import "UIImageView+AFNetworking.h"
 
+static NSMutableArray *teamsInTournament;
```

Comment:
any reason to make this static const? i think this makes more sense as a property on HomeViewController